### DISCUSSION TITLE
PCB für onBattery?

### DISCUSSION THREAD

**Discussion by Adminius** [hoylabs/OpenDTU-OnBattery#148] [General]
Hallo zusammen,

wie viele von uns nutze ich eine Prototype-PCB für OpenDTU-onBattery.
Es sind schon mehrmals schlechte Lötstellenstellen aufgefallen und allgemein gefällt mir es nicht.

Ich habe ganz rudimentäre Erfahrung in PCB Layouten und könnte ein paar in China fertigen lassen (nur die PCBs ohne Bestückung)

Meine Idee ist um die Sache, einfach zu gestallten, fertige Komponenten von Aliexpress, wie ESP, CAN PCB, RS485 usw. einfach steckbar machen. Dann kann jeder bestücken wie er will.

Folgende Sachen würden ungefähr aktuellem DEV Branch entsprechen (Links sind Beispiele):

- ESP32 mit USB-UART-Chip und MicroUSB/USB-C für die Stromversorgung: https://www.aliexpress.com/item/1005004879572949.html
- NRF24L01+ Teil (Mit Antenne oder ohne oder beide Möglichkeiten?!) z.B https://www.aliexpress.com/item/32848834985.html
- CAN + RJ45 Buchse+Schraubklemme https://www.aliexpress.com/item/32375478262.html
- SPI CAN MCP2515 + Schraubklemme https://www.aliexpress.com/item/32799974260.html
- 2xRS485 + Schraubklemme https://www.aliexpress.com/item/1005001621746811.html
- 3-4x ADUM1201 + Schraubklemmen oder fertige JSTs z.B. für Victron https://www.aliexpress.com/item/1005003649567525.html
- Pinheader für i2c Display
- Tasten?

Was noch? Relais? Bzw. einfach ein paar Schraubklemmen für weitere IOs?

Statt Schraubklemmen kann man auch Federklemmen nehmen:
https://www.aliexpress.com/item/1005002475427338.html
https://www.aliexpress.com/item/1005003102030062.html

Bei ADUM denke ich an die Victrons (5V->3.3V)
aber auch ans BMS um die GND von dem BMS von ESP32 zu trennen. Hier ist aber das Problem, dass manche BMS entweder volle VBAT Spannung mitliefern, RX/TX sind aber bei ca. 3V (z.B. JK BMS) oder gar keine (JBD), RX/TX sind auch bei 3.3V. An der Stelle ist es am einfachsten mit isoliertem DC-DC zu arbeiten B0503S-1W:  https://www.aliexpress.com/item/1005004891463375.html

An sich wird es eine für OpenDTU angepasste https://github.com/shining-man/bsc_fw sein.

Dazu noch 3D gedrucktes Gehäuse oder auch etwas fertiges...
Was haltet ihr davon?

**Comment by swingstate** [comment]
Eine gute Idee! Könnte man die verschiedenen Spannungen für die diversen BMS Optionen über ein Dip Switch einstellen? 

Vielleicht noch etwas Inspiration: https://github.com/dokuhn/openDTU-BreakoutBoard/discussions/15#discussioncomment-5397056

**Reply by Adminius** [reply]
DIP Switch ist wieder komplex.
Meine Idee ist ja, dass die benötigte Spannung auf der BMS Seite des ADUMs, also 5 oder 3,3V aus den 5V des ESP32/USBs genommen werden.
Da gibt es entsprechnde isonlierte DC-DC Wandler, in 5->5V oder 5->3.3V.
Mehr als 5V kann der ADUM sowieso nicht vertragen und würde es noch mal komplexer machen. Die Frage ist ob es auch nötig wäre eine andere Spannung als 5 oder 3.3V zu haben? Ich kenne keinen BMS der auf RX/TX weder 5 noch 3.3 haben will.
Victrons sind fast alle 5V.

D.h. wenn man die Platine aufbaut, muss man selbst wissen welche Spannungswandler genommen werden müssen.
Danke für die Inspiration, schaue ich mir genau an. Das ist aber komplett anderer Ansatz: wirklich alles verlöten. Das schafft leider nicht jeder, vor allem QFN Package.
Ich habe eher an sowas gedacht nur größer und "mächtiger":
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/138

**Reply by swingstate** [reply]
Verstehe - Dein Ansatz bzgl. DC-DC macht Sinn. Die von Dir verlinkten Produkte müssen aber auch noch z.t. verlötet werden, oder (Stiftleisten)? Was Du vorschlägst geht ja in die Richtung des [Grove Systems](https://www.seeedstudio.com/category/Grove-c-1003.html) von Seeed. Das wäre schon cool sowas für die DTU zu entwickeln.

**Reply by Adminius** [reply]
ja gut, ein wenig löten muss man ja, aber THT ist für die meisten OK (ich bin eher der Freund von "QFN händisch löten" 🤣 )
Stiftleisten und die Aufnahmen dazu, DC-DC Wandler, Klemmen, das muss alles gelötet werden, evtl. ein paar Kondensatoren auch in THT. 
Falls man gar nicht löten kann, ist man eh im falschen Projekt ;)
Ein Multimeter wäre auch von Vorteil. Ansonsten kann sich jemand aus der Community dafür engagieren und den Leuten die es nicht haben/können helfen. 

Alternative wäre bei den Chinesen auch gleich bestücken zu lassen, da bin ich aber raus, keine Zeit mich darum zu kümmern, sorry.
Da sind dann aber die Preise auch nicht mehr so attraktiv, und wo bleibt der Spaß am Basteln? ;)

**Reply by MalteSchm** [reply]
Ich halte da viel von :-)

Ein Punkt der mir noch einfällt: 
Ein 3.3V - 5V pegelshifter und ein 5V Relais. Letzteres benötigt man um den Slot Detect Eingang dieses Huawei Ladegerät zu schalten. Ohne das läuft der Lüfter durch. 

Wenn du da welche extra machen lässt: Ich würde so eine Platine nehmen

Den Ansatz fertige Module zu nutzen halte ich für richtig

**Reply by swingstate** [reply]
Aus Mangel an Fähigkeiten - Könnte man sich nicht einen Platine designen lassen, beispielsweise über fiverr.com? Und dann bei Aisler ordern? 
Mir schwebt ein Kästchen vor dass die Pylontech (oder besser jede andere LifePo) anschliessen kann um die Speicherung von Strom zu steuern und auf der Stromeingangsseite Flexibel ist und 1 - 2 MPPTs oder das Huawei NT "Plug'n'Play" akzeptiert. 

too much?

**Reply by madmartin** [reply]
> Aus Mangel an Fähigkeiten - Könnte man sich nicht einen Platine designen lassen, beispielsweise über fiverr.com? Und dann bei Aisler ordern? 

Das kann teuer werden. Besser jemand mit entsprechenden Fähigkeiten macht ein Layout mit KiCad, veröffentlicht das als Github Projekt (ähnlich wie https://github.com/SteffMUC/openDTU_wDisplay2 ) und dann kann man das mit Hilfe der Gerber-Dateien bei z.b. jlpcb.com bestellen. Jeder wie er mag, indiviuell oder als Sammelbestellung. Oder die Platine nach eigenen Wünschen anpassen.

> Mir schwebt ein Kästchen vor dass die Pylontech (oder besser jede andere LifePo) anschliessen kann um die Speicherung von Strom zu steuern und auf der Stromeingangsseite Flexibel ist und 1 - 2 MPPTs oder das Huawei NT "Plug'n'Play" akzeptiert.
> 
> too much?

Nicht unbedingt too much, aber bitte bedenke daß die PylonTech Akkus ziemlich gute Sicherheitseinrichtungen drin haben (Abschaltung bei Unterspannung, Soft-Start), was viele andere nicht haben, das schliesst dann "jede andere LiFePo" eigentlich aus.
Man sollte sich da schon etwas fokussieren.

>Den Ansatz fertige Module zu nutzen halte ich für richtig

Ich auch.

Schon mal #138 angesehen?

**Reply by Adminius** [reply]
mit KiCad kenne ich mich leider nicht aus und habe jetzt keine Zeit mich einzuarbeiten.
Ich werde es mit Eagle machen, da gibt es auch Freie Version. 
Open Source ist ja gesetzt. Gerber Files wird es geben.
Man muss aber eine Sache bedenken, meistens werden günstige Platinen nur bis zu einer bestimmter Größe gefertigt, ca. 10x10cm wenn ich mich nicht täusche. Wegen fertigen Module bleibt auch nicht viel Platz übrig. 
Ich schau was möglich ist und die Zeit dafür finden, aktuell gar nicht so einfach und es wird nur schlimmer...

Bzgl. Relais: Ich bin kurz davor zu sagen, Relais ist so ein spezifischer Fall, dass es besser ist ein Paar IOs dafür vorzusehen und ein fertiges Relais-Modul von Ali zu nehmen.
Und muss es wirklich ein Relais sein? Die fressen ganz schön viel, wegen einem Lüfter ist es echt ineffizient. Reicht da nicht ein Optokoppler aus? Oder gar eine direkt IO-Ansteuerung?
Ich weiß auch ehrlich gesagt nicht wir viele so einen AC-Laderegler wirklich einsetzen werden...

Was bringt 3.3V-5V Pegelwandler? Will nur verstehen...

Bzgl. anderen LiFePos. ich habe jetzt einen JK-BMS, der auch viele Sicherheitseinrichtungen hat und per UART sprechen kann. Diesen werde ich einbauen sobald andere Sachen gelöst sind. Ich hätte noch JBD (sehr günstig) der kann auch per UART sprechen, aber sehr wenig Doku dafür. Es bleibt dann "nur noch" DALI BMS übrig. Alles 3 können bei Temperatur, over/undervoltage, overcurrent abschalten. Das reicht für unsere Anwendung aus.

**Reply by MalteSchm** [reply]
Der Pegelwandler ist für Peripherie die 5V benötigt. Es wird sowas geben. IOs mit 5V würden dann auch für das Relais ausreichen. Ich bin auch in jedem Fall von einem fertigen Relaismodul ausgegangen. So wie das gerade gelöst ist wird das nur geschaltet wenn auch das Netzteil (mit dann immerhin bis zu 3kw) läuft. Da ist der Stromverbauch für so ein Relais auch egal.

Bzgl. AC Netzteil Einsatz und Features: wird man sehen. Man muss bei einer solchen Platine ja nicht alles bestücken.

**Reply by Adminius** [reply]
ich schau mal was möglich ist.
Bei fertigen Modulen von Ali gibt's noch anderes Problem: Da kenne ich die Abmessungen meistens nicht und muss mir dafür eigene Bibliotheken erstellen, dabei am Besten die Module auf dem Tisch haben...
Bei SMD Komponenten ist es einfach, entweder gibt es schon etwas oder Datenblatt lesen ;)

**Reply by madmartin** [reply]
>  Bei fertigen Modulen von Ali gibt's noch anderes Problem: Da kenne ich die Abmessungen meistens nicht und muss mir dafür eigene Bibliotheken erstellen, 

Oder du kaufst bei einem Lieferanten hier, der einigermassen zuverlässig ist, und doch preiswert, so empfinde ich z.b. AZ-Delivery als einen solchen.
Meine Erfahrungen mit Ali-Express Lieferanten und dem Versand sind /sehr/ durchwachsen. Das meide ich wenn immer möglich und ich es mir leisten kann.

**Reply by Adminius** [reply]
mir geht es nicht darum wo ich kaufe, sondern dass ich kaufen muss um nach zumessen, egal ob ich es brauche oder nicht...

**Reply by swingstate** [reply]
Frage an jemanden mit Erfahrung im PCB design - ist so eine Platine rein konzeptionell denkbar, oder out of range? 

Eine Kombination aus: 

1. DTU / ESP32 + NRF+ + optinales Display
2. 2x Ve. direct Verbindungen zu den MPPTs
3. Can Bus zur Batterie
4. Inverter Precharge bzw. Currentlimiter und DC-seitigem Switch

<img width="675" alt="Bildschirmfoto 2023-04-04 um 22 03 58" src="https://user-images.githubusercontent.com/73672554/229908015-11ca1d87-e2c8-4f8b-a64d-2c19df2879c9.png">

**Reply by Adminius** [reply]
Hi, 
ich würde es so nicht machen:
- Du willst User-Intarface mit Tasten auf die selbe Platine mit 20-30A Strom und bis zu 60V freiliegende Spannung -> Keine gute Idee
- D.h. Display+Tasten müsste man auf die 2. Platine auslagern, so ist auch ein Gehäuse dafür zu bauen/finden
- SMA + NRF gehören zusammen.
- Batterieleitungen über die PCB jagen, mit 20-30A (beim kleinen Balkonkraftwerk) ist schon herausfordern wegen dicken Leiterbahnen, wenn jemand noch mehr Ströme anschließen wird...-> lieber nicht, da würde ich auf etwas fertiges setzen

Ich verstehe zwar den Ansatz, aber aber die Platine wird riesig und nicht flexibel, oder ist nur für einen konterten Fall gedacht?

**Reply by berni2288** [reply]
Hi,
ihr fackelt euch euer ganzes Haus ab wenn ihr soviel Ampere über PCB-Leitungen jagen wollts, damit ist echt nicht zu spaßen.

HIer ein paar Anregungen weil ich nicht will dass euch etwas passiert:
- Wenn man eine Batterie benutzt ist es absolut unerlässlich zu wissen wie groß der Querschnitt der Leitungen sein muss um diese hohen Amperezahlen zu bewältigen
- Man sollte wissen wie man die Leitungen voneinander abschirmt und abdeckt damit es nicht zu unabsichtlichen Kurzschlüssen beim Arbeiten oder auch unbeaufsichtigt kommt.
- Bei zu losen Kontakten an den Kabelenden kann es ebenfalls zu Bränden kommen
- Man sollte schon etwas Ahnung von Elektrotechnik & Elektronik haben. Ohmsches Gesetz, Widerstände, Leiterquerschnitte etc.

Bitte bastelt nichts zusammen wo ihr euch nicht sicher seid dass es "safe" ist das unbewacht zu betreiben. Vor allem nicht Zuhause.
Ein Brand kann schneller da sein als ihr denkt, und das kann passieren wenn ihr grad nicht zuhaus seid oder um 03:00 in der Früh.

Danke!

**Reply by madmartin** [reply]
Zustimmend zu @berni2288 empfehle ich all denen die nicht täglich mit Volt und Ampere zu tun haben oder es mal als Beruf gelernt haben dieses e-Book:
https://www.victronenergy.com/upload/documents/The_Wiring_Unlimited_book/43562-Wiring_Unlimited-pdf-en.pdf
https://www.victronenergy.com/upload/documents/The_Wiring_Unlimited_book/43562-Wiring_Unlimited-pdf-de.pdf

Zum Thema Volt ist auch noch anzumerken: ein Stromschlag bei 230V Wechselspannung ist sehr unangenehm, aber je nach Situation nicht lebensgefährlich.
Ein Stromschlag von nur 80-100V Gleichspannung (z.B. 2 große Solarpanel in Reihe geschaltet) sind durchaus LEBENSGEFÄHRLICH. Bei Gleichspannung verhält sichder Körper vollkommen anders als Wechselspannung. Das muß jedem klar sein der damit rumhantiert.

Und nicht zuletzt: Akkus haben sehr hohe Kurzschlußströme. Ich habe schon einen glühenden Schraubendreher gesehen.

**Comment by modem-man-gmx** [comment]
Vielleicht hängt Ihr Euch ja an Markusdd und sein Ahoy/openDTU Board-Design ran?

wird in einem [Discord Channel](https://discord.com/channels/984173303147155506/1060347063281401876) besprochen. Bin gerade dabei, meins zum laufen zu bringen (soll heissen: das Board existiert schon, nur die Batteriesteuerung müsste noch dran)

![image](https://user-images.githubusercontent.com/96957387/230298665-25bf529f-280e-422b-b978-674c746138b0.png)

**Reply by Adminius** [reply]
Nein, obwohl das Board cool aussieht :)
Sein Board ist für einen Leie nicht lötbar.
Mein Ansatz ist ja: fertige Module nehmen, die zusammenstecken, läuft (ok, die Pinheader + Klemmen müsste man löten, das schaffen schon die meisten)

**Reply by madmartin** [reply]
> Nein, obwohl das Board cool aussieht :) Sein Board ist für einen Leie nicht lötbar. Mein Ansatz ist ja: fertige Module nehmen, die zusammenstecken, läuft (ok, die Pinheader + Klemmen müsste man löten, das schaffen schon die meisten)

Volle Zustimmung.

**Comment by modem-man-gmx** [comment]
Anm.: Das Bild ist V2, das existiert noch nicht.

**Comment by Tyberyus** [comment]
Währe es möglich noch einen Klinken Eingang mit rein zu nehmen um eventuell einen externen Stromsensor wie den STC-13 anstecken zu können? Damit könnte man dann auch das PowerLimit steuern.

**Reply by Adminius** [reply]
bei meinem Prototypen habe ich sogar 3 stück gehabt, aber die Messung war sehr ungenau, so um +/- 50W, d.h. komplett unbrauchbar bei unseren Leistungen.

**Comment by modem-man-gmx** [comment]
Sofern ich es richtig überblicke, ist das Hauptproblem bei der Nutzung einer Batterie+Laderegler am Solarpanel-Eingang, dass die Eingangsschaltung der Wechselrichter - insbesondere die großen Kondensatoren - nicht dafür ausgelegt sind plötzlich mit nahezu unbegrenztem Strom geladen zu werden und vielleicht nach wenigen solcher Schaltvorgänge die Grätsche machen. Da hilft bestenfalls das netzseitige Trennen des WR über Tags und netzseitige Zuschalten sobald es "dunkel" wird. In diesem Fall läge am WR-Eingang immer die Batteriespannung schon an, die Kondensatoren wären im Leerlauf, aber voll.

Korrekt soweit?

Das ganze ist zwar so handhabbar, aber nicht ausfallsicher:
* Nimmt man einen Dämmerungsschalter, kann der auch schonmal zum falschen Zeitpunkt auslösen
* Schaltuhren ohne DCF77 können komplett falsch gehen
* der WR "nuckelt" im Winter die Batterien leer

Eine Lösung wäre also, 
* die WR-Eingänge immer etwas vorzuspannen (das nuckelt aber weiter...) oder
* die WR-Eingänge langsam hochzufahren (dicken FET davor?)

Da bleibt für mich die Frage: Warum nicht eine niederohmige, hohe Induktivität an den Eingang? einfach 30 Windungen 2,5² auf einen Ringkern müßte doch in etwa den selben Effekt wie die FET-Lösung haben?

**Reply by Adminius** [reply]
Warum zu und abschalten? 
Der WR verbraucht im Stanby 1,7W auf der DC Seite. 
Aus aktueller Erfahrung: 810 Wp Süd, 90° auf Balkongeländer. Regnerisches Wetter: 60-70W PV Leistung. Da wird nicht nur Standyby verbrauch gedeckt, sondern auch Akku sehr langsam nachgeladen.
1,7W * 16h (Winterliche Nacht + dunkle Stunden) => 27Wh da hat man in täglich drin um in den Akku zu laden und den nicht leer zu saugen. Ansonsten Einstellungen so verändern, dass der Akku voller gehalten wird. Genauer kann ich dann erst im Januar/Februar sagen.

**Reply by modem-man-gmx** [reply]
Okay, 2W * 24h Standby des WR gegen 6h * 20W PV, das passt. 
Mit An/Abschalten meine ich den Ausgang des WR. Kann hier etwas missverständlich gewesen sein, weil ich gleichzeitig den Vorschlag einiger Projekte erwähnt habe, den WR-Eingang per MOSFET vom Akku wegzuschalten.

Das Abschalten des WR-230V Ausgangs muss m.E. unbedingt sein, denn wenn im Winter weniger vom Panel reinkommt, als die Nennleistung am WR-Ausgang ist, zieht der die Batterien zu tief. Oder klemmt man den WR nicht direkt an die Batterien, sondern an einen smart geschalteten Ausgang des Laderegler??

Und was ich mich noch frage: 
wenn der WR die ganze Zeit am Akku hängt, dann läuft doch die MPP-Regelung des WR gegen den Ausgang des Ladereglers, der seinerseits eigenes MPP gegen die Module macht. Fängt das nicht an zu schwingen?

:-?

**Reply by Tyberyus** [reply]
Also ich plane den Strom mittels einer simplen Schaltung mit einem TIP35 auf 11,5A zu begrenzen und dann ist es egal ob mal die Spannung von der Batterie weg bricht. 

[](https://www.electronics-notes.com/images/regulator-voltage-emitter-follower-with-current-limit-01.svg)

Eventuell sollte das gleich mit auf der Platine vorgesehen werden? Sind 6 Bauteile mehr für jeden Eingang eines MicroWR. 
Zusätzlich vielleicht auch noch die Möglichkeit einen externen Step Up Converter anzuschließen der die Spannung zuvor anpasst. Ist z.B. für 12v Systeme relevant. Oder dafür lieber eine extra Platine?

**Reply by Tyberyus** [reply]
Hab gerade mal nachgerechnet mit der Schaltung hätte man mit rund 8W an Verlusten zu rechnen. Vielleicht nicht ideal. Dann lieber einen MOSFET der einen parallel liegenden Widerstand überbrückt. 
![image](https://user-images.githubusercontent.com/49712087/234196788-a799f242-4d7c-4369-8abb-1d6c4606a21b.png)

**Reply by Adminius** [reply]
Bei 8 W kann man gleich einen Contactor nehmen. da Verbraucht die Spule so viel.
Aber auf deinem Bild überbrückt Q1 den R1, was willst du da anders machen?
Evtl. müsste man negativen Pol und nicht Positiven trennten.

So oder so, mein Ziel ist erst mal OpenDTU auf eine Platine zu bringen. Strombegrenzung sehe ich optional -> extra Platine. Und zwar eine pro WR Kanal, dann kann jeder selber schauen wie viele er braucht. Für meinen HM350 brauche ich keine 2 Kanäle ;) 

> Das Abschalten des WR-230V Ausgangs muss m.E. unbedingt sein, denn wenn im Winter weniger vom Panel reinkommt, als die Nennleistung am WR-Ausgang ist, zieht der die Batterien zu tief.

Das erledigt OpenDTU-onBattery per Software Steuerung.


> wenn der WR die ganze Zeit am Akku hängt, dann läuft doch die MPP-Regelung des WR gegen den Ausgang des Ladereglers, der seinerseits eigenes MPP gegen die Module macht. Fängt das nicht an zu schwingen?

weder mein HM600 noch HM350 suchen nach MPP wenn die am Akku hängen. Es kann an 15S Batterie und hoher 45+ Volt Spannung liegen. aber als ich mit 12S experimentiert habe habe ich auch nichts festgestellt. 
Ich werde in ein paar Monaten meine 15S Batterie auf 12S2P umbauen (ich habe noch 8S nachgekauft und 1 Zelle liegt nur rum). Dann werde ich noch mal schauen ob MPP Probleme macht. Das ist aber hier Offtopic.

**Reply by Adminius** [reply]
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/145#discussioncomment-5718624

**Reply by modem-man-gmx** [reply]
> Hab gerade mal nachgerechnet mit der Schaltung hätte man mit rund 8W an Verlusten zu rechnen. Vielleicht nicht ideal. Dann lieber einen MOSFET der einen parallel liegenden Widerstand überbrückt. ![image](https://user-images.githubusercontent.com/49712087/234196788-a799f242-4d7c-4369-8abb-1d6c4606a21b.png)

Huh, obwohl ich seit 25 Jahren keine analogen Schaltungen mehr angeschaut habe, fällt mir der "falsch herum" eingezeichnete Transistor unglaublich schwer zu verstehen. Es sträubt sich alles in mir, die Schaltung zu begreifen. 
:-O

**Comment by swingstate** [comment]
Alternative Idee bzw. so wie es mache, da mein Batterie WR nicht an einem DC Ladereglern hängt - Einfach ein kleines AC Ladegerät mit an die Batterie hängen und so ggfs. sicherstellen, dass diese nicht zu tief entladen wird. 

In eurem Fall könnte das ein Fallback sein, in meinem Fall ist das die Standardconfig. Um den WR nicht zu killen kann man auch den inrush current limitieren - Details [hier](https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/145) @modem-man-gmx

**Comment by MalteSchm** [comment]
Hallo,

ist vielleicht nicht bekannt:
in dem Docs folder ist eine Fritzing Datei die die Grundlage einer Platine darstellen könnte. Ich habe eben nur mal schnell Bauteile platziert und den Autorouter angeworfen:

![image](https://user-images.githubusercontent.com/88371275/234084742-7e0a58bd-4157-4010-b265-461a1ed6376e.png)

**Reply by madmartin** [reply]
ich finde KICAD viel schöner...

**Comment by Adminius** [comment]
ich bin noch dran... ;)
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/17052262/3193e8af-671e-4f99-9647-02b2c033cd09)

**Comment by swingstate** [comment]
ich auch :) 

momentan in Arbeit - 4x inrush protected DC channel bis 11,5A, support für 24 - 48V, Relays auf AC und ggfs. DC Seite - mal schauen ob's was wird.
